test/scan_window_test.dart

```diff
@@ -152,6 +154,191 @@ void main() {
       });
     },
   );
+
+  group(
+    'Calculate Box Fit Ratio',
```

Comment:
Can we use the name of the function under test as the topmost group name?

```suggestion
    'calculateBoxFitRatio',
```